SitecoreDeployer/buildtask/functions-module-v1.psm1

```diff
@@ -161,7 +161,7 @@ function CollectOutBoundIpAddresses {
 
     $collectedIps = "";
     #Get all resources, which are in resource groups, which contains our name
-    $resources = Find-AzureRmResource -ResourceGroupNameContains $resourceGroupName
+    $resources = Get-AzureRmResource -ResourceGroupName *$resourceGroupName*

```

Comment:
Please, remove asteriks here - we need resources from specific resource group, not matching one (in Find-AzureRmResource there was not possibility to match exact rg name)